helpers.sh

```diff
@@ -32,7 +32,18 @@ function repo_sync {
 		git config --global http.https://${domain}.extraheader "$(cat /secrets/git.http.extraheader)"
 	fi
 	run repo init --no-clone-bundle -u $* ${REPO_INIT_OVERRIDES}
-	run repo sync
+	for i in $(seq 4); do
+		run timeout 5m repo sync && break
+		if [ $? -eq 124 ] ; then
+			msg="Command timed out"
+			if [ $i -ne 4 ] ; then
+				msg="${msg}, trying again"
+			fi
+			status Command timed out, trying again
```

Comment:
I guess it should be `status ${msg}`

Reply:
Yeah, not sure why we need this additional if if 124 means a timeout, you could just have the status line as it will be aborted correctly coming from the previous for.